test/0.4.24/contracts/StakingRouter__MockForLidoAccounting.sol

```diff
@@ -3,8 +3,11 @@
 
 pragma solidity 0.8.9;
 
+import {StakingRouter} from "contracts/0.8.9/StakingRouter.sol";
+
 contract StakingRouter__MockForLidoAccounting {
```

Comment:
Noticed that this file is being used in the accounting unit tests. It would be better to clone the file and rename it to StakingRouter__MockForLidoAccountingFuzzing to clarify its purpose. Separating the mocks will simplify mock management and ensure that they do not interfere with other tests.

Reply:
fixed